reco_utils/setup.py

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+from setuptools import setup, find_packages
+from os import path
+
+
+VERSION = __import__('__init__').VERSION
+
+here = path.abspath(path.dirname(__file__))
+
+# Get the long description from the README file
+with open(path.join(here, 'README.md'), encoding='utf-8') as f:
+    long_description = f.read()
+
+setup(
+    name='reco_utils',
+    version=VERSION,
+    description='Recommender System Utilities',
+    long_description=long_description,
+    long_description_content_type='text/markdown',
+    url='https://github.com/microsoft/recommenders',
+    author='RecoDev Team at Microsoft',
+    author_email='RecoDevTeam@service.microsoft.com',
+    classifiers=[
+        'Development Status :: 4 - Beta',
+        'Intended Audience :: Developers',
+        'Intended Audience :: Science/Research',
+        'Topic :: Scientific/Engineering',
+        'Topic :: Software Development :: Libraries :: Python Modules',
+        'License :: OSI Approved :: MIT License',
+        'Programming Language :: Python :: 3',
```

Comment:
in the conda package we have 3.6.8 https://github.com/microsoft/recommenders/blob/master/scripts/generate_conda_file.py#L49 but here we are saying that we support 3.4-3.7. do we know if all recommenders will work on all these versions?

Reply:
hmm, yeah fair enough. I'll set it to 3.6, we can add additional versions as we test them and ensure they work.